crates/matrix-sdk-sqlite/src/utils.rs

```diff
@@ -22,6 +22,8 @@ use matrix_sdk_store_encryption::StoreCipher;
 use ruma::time::SystemTime;
 use rusqlite::{limits::Limit, OptionalExtension, Params, Row, Statement, Transaction};
 use serde::{de::DeserializeOwned, Serialize};
+#[cfg(not(test))]
+use tracing::warn;
```

Comment:
I'd use `tracing::warn!` at the single place where it's used instead, to not avoid a conditional `use` statement (which I personally find ugly).

Reply:
Done